packages/controller-utils/src/util.test.ts

```diff
@@ -11,6 +12,11 @@ describe('util', () => {
     nock.cleanAll();
   });
 
+  it('isSafeChainId', () => {
+    expect(util.isSafeChainId(MAX_SAFE_CHAIN_ID + 1)).toBe(false);
+    expect(util.isSafeChainId(MAX_SAFE_CHAIN_ID)).toBe(true);
```

Comment:
Maybe a couple more tests for the `isSafeInteger` and `> 0` checks? 

``` suggestion
    expect(util.isSafeChainId(MAX_SAFE_CHAIN_ID + 1)).toBe(false);
    expect(util.isSafeChainId(MAX_SAFE_CHAIN_ID)).toBe(true);
    expect(util.isSafeChainId('not-a-number')).toBe(false);
    expect(util.isSafeChainId(-1)).toBe(false);
```

Reply:
done here.
https://github.com/MetaMask/core/pull/1064/commits/a61da912d40b3bd9276f3f34abe97e057690324e

> expect(util.isSafeChainId('not-a-number')).toBe(false);

typescript won't let me add this test since `isSafeChainId` arg is typed as a string

Reply:
Oh, right, makes sense. What if we use a `@ts-expect-error` line above it? I think that would be okay in this case.